### PR TITLE
[Merged by Bors] - chore: tidy various files

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Algebra/BigOperators/Group/List.lean [hunk 1]
```diff
@@ -432,9 +432,9 @@ lemma prod_rotate_eq_one_of_prod_eq_one :
   | [], _, _ => by simp
   | a :: l, hl, n => by
     have : n % List.length (a :: l) ≤ List.length (a :: l) := le_of_lt (Nat.mod_lt _ (by simp))
-    rw [← List.take_append_drop (n % List.length (a :: l)) (a :: l)] at hl;
```
Comment (grunweg) [inline Mathlib/Algebra/BigOperators/Group/List.lean:435]: And I thought I had caught all such lines at one point - thanks for fixing this!

File: Mathlib/Analysis/NormedSpace/Star/ContinuousFunctionalCalculus/Order.lean [hunk 2]
```diff
@@ -100,9 +100,8 @@ lemma IsSelfAdjoint.toReal_spectralRadius_eq_norm {a : A} (ha : IsSelfAdjoint a)
 lemma CstarRing.norm_or_neg_norm_mem_spectrum [Nontrivial A] {a : A}
     (ha : IsSelfAdjoint a := by cfc_tac) : ‖a‖ ∈ spectrum ℝ a ∨ -‖a‖ ∈ spectrum ℝ a := by
   have ha' : SpectrumRestricts a Complex.reCLM := ha.spectrumRestricts
-  have := ha.toReal_spectralRadius_eq_norm
-  convert Real.spectralRadius_mem_spectrum_or (ha'.image ▸ (spectrum.nonempty a).image _)
-    <;> exact id (Eq.symm this)
+  rw [← ha.toReal_spectralRadius_eq_norm]
```
Comment (grunweg) [inline Mathlib/Analysis/NormedSpace/Star/ContinuousFunctionalCalculus/Order.lean:103]: Nice one!

File: Mathlib/MeasureTheory/Measure/Lebesgue/VolumeOfBalls.lean [hunk 3]
```diff
@@ -198,9 +198,7 @@ theorem MeasureTheory.volume_sum_rpow_lt_one :
 theorem MeasureTheory.volume_sum_rpow_lt [Nonempty ι] {p : ℝ} (hp : 1 ≤ p) (r : ℝ) :
     volume {x : ι → ℝ | (∑ i, |x i| ^ p) ^ (1 / p) < r} = (.ofReal r) ^ card ι *
       .ofReal ((2 * Gamma (1 / p + 1)) ^ card ι / Gamma (card ι / p + 1)) := by
-  have h₁ : ∀ x : ι → ℝ, 0 ≤ ∑ i, |x i| ^ p := by
-      refine fun _ => Finset.sum_nonneg' ?_
-      exact fun i => (fun _ => rpow_nonneg (abs_nonneg _) _) _
+  have h₁ (x : ι → ℝ) : 0 ≤ ∑ i, |x i| ^ p := by positivity
```
Comment (grunweg) [inline Mathlib/MeasureTheory/Measure/Lebesgue/VolumeOfBalls.lean:201]: Nice small golf!

File: Mathlib/MeasureTheory/Measure/Lebesgue/VolumeOfBalls.lean [hunk 4]
```diff
@@ -273,9 +271,7 @@ theorem Complex.volume_sum_rpow_lt_one {p : ℝ} (hp : 1 ≤ p) :
 theorem Complex.volume_sum_rpow_lt [Nonempty ι] {p : ℝ} (hp : 1 ≤ p) (r : ℝ) :
     volume {x : ι → ℂ | (∑ i, ‖x i‖ ^ p) ^ (1 / p) < r} = (.ofReal r) ^ (2 * card ι) *
       .ofReal ((π * Real.Gamma (2 / p + 1)) ^ card ι / Real.Gamma (2 * card ι / p + 1)) := by
-  have h₁ : ∀ x : ι → ℂ, 0 ≤ ∑ i, ‖x i‖ ^ p := by
-      refine fun _ => Finset.sum_nonneg' ?_
-      exact fun i => (fun _ => rpow_nonneg (norm_nonneg _) _) _
+  have h₁ (x : ι → ℂ) : 0 ≤ ∑ i, ‖x i‖ ^ p := by positivity
```
Comment (grunweg) [inline Mathlib/MeasureTheory/Measure/Lebesgue/VolumeOfBalls.lean:274]: Here too!

**Review by grunweg — COMMENTED** [review]
Looks great - thanks for doing this!
maintainer merge

**Comment by riccardobrasca** [comment]
Thanks!

bors merge

**Pull request closed without merging**